pkg/spdk/disk.go

```diff
@@ -85,25 +85,31 @@ func svcDiskDelete(spdkClient *spdkclient.Client, diskName, diskUUID, diskPath,
 		}
 	}()
 
-	if diskName == "" || diskUUID == "" {
-		return &emptypb.Empty{}, grpcstatus.Error(grpccodes.InvalidArgument, "disk name and disk UUID are required")
+	if diskName == "" {
```

Comment:
What if somehow only `diskUUID` is provided? 
The function should try its best to do the cleanup:
1. If `diskName` is provided, we will use this name as lvstore name during deletion
2. If `diskUUID` is provided, we will use this UUID as lvstoreUUID during deletion 
3. If both are provided, we will verify there is an exact lvstore matching both during deletion

Reply:
> What if somehow only diskUUID is provided?

Only `diskUUID` should be impossible, because the diskName is must-have information, so we only need to handle empty UUID case.
Ref: https://github.com/longhorn/longhorn-manager/blob/master/controller/node_controller.go#L1680-L1698